src/app.js

```diff
@@ -0,0 +1,64 @@
+import "regenerator-runtime/runtime";
```

Comment:
What's this import for?

Reply:
I did not add this line, I think that it was inserted by default wit he npm configuration. But what I read was that it is because the compiler that I am using is babel and after it makes syntax transformation the resulting code relies on runtime support from this regenerator-runtime.

Reply:
:open_mouth: